Formula/safe.rb

```diff
@@ -0,0 +1,40 @@
+class Safe < Formula
+  desc "Vault CLI"
+  homepage "https://github.com/starkandwayne/safe"
+  url "https://github.com/starkandwayne/safe/archive/v0.8.0.tar.gz"
+  sha256 "0d592e7689e7c9d729f69671d3afa0b4400626fb025d56af5b857345d1310dbf"
```

Comment:
Need a newline between this & the first `depends_on`.

Reply:
fixed!  is this style requirement documented somewhere?